guides/howto/custom_error_pages.md

```diff
@@ -96,6 +96,14 @@ Phoenix generates an `ErrorView` for us, but it doesn't give us a `lib/hello_web
 </html>
 ```
 
+Additionally, lets remove the `render/2` function we previously introduced in `lib/hello_web/views/error_view.ex`.
```

Comment:
```suggestion
After you define the template disk, remember to remove the equivalent `render/2` clause for that template, as otherwise the function overrides the template. Let's do so for the 404.html clause we have previously introduced in `lib/hello_web/views/error_view.ex`:
```

Reply:
template disk?
"template file" maybe?